### ISSUE TITLE
VPCS 0.6.1 show arp will cause exception if arp table is not empty

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-server#355]
I realize VPCS is not GNS3, however it is included with GNS3 so I though I should note this.
 
As a work around I recommend including VPCS 0.6 with future GNS3 builds. The following terminal session exemplifies the issue. Notice the text in bold shows the first "show arp" command succeeds when the arp table is empty. The next attempt fails when the arp table is not empty.
 
```
-bash$ ./vpcs
 
Welcome to Virtual PC Simulator, version 0.6.1
Dedicated to Daling.
Build time: Jun  1 2015 10:26:51
Copyright (c) 2007-2014, Paul Meng (mirnshi@gmail.com)
All rights reserved.
 
VPCS is free software, distributed under the terms of the "BSD" licence.
Source code and license can be found at vpcs.sf.net.
For more information, please visit wiki.freecode.com.cn.
 
Press '?' to get help.
 
VPCS[1]> ip 10.1.1.1/24
Checking for duplicate address...
PC1 : 10.1.1.1 255.255.255.0
 
VPCS[1]> 2
VPCS[2]> ip 10.1.1.2/24
Checking for duplicate address...
PC2 : 10.1.1.2 255.255.255.0
 
VPCS[2]> set lport 30000
 
VPCS[2]> set rport 20000
 
VPCS[2]> show arp
 
arp table is empty
 
VPCS[2]> ping 10.1.1.1
84 bytes from 10.1.1.1 icmp_seq=1 ttl=64 time=0.209 ms
84 bytes from 10.1.1.1 icmp_seq=2 ttl=64 time=0.237 ms
^C
VPCS[2]> show arp
 
Abort trap: 6
-bash$
```

**Comment by pubyun** [comment]
+1

**Comment by grossmj** [comment]
This should be reported to Paul, VPCS developer.

**Comment by noplay** [comment]
It's already reported no news for that point.

**Comment by noplay** [comment]
Full session to replicate:
```
ip 10.1.1.1/24
2
ip 10.1.1.2/24
set lport 30000
set rport 20000
ping 10.1.1.1
show arp

**Comment by noplay** [comment]
Mail send to VPCS author

**Comment by noplay** [comment]
Seen with him it's an issue with LLVM

**Comment by noplay** [comment]
New version will be shipped with 1.4.5 on Mac

**Comment by VNVetrov** [comment]
I have exactly the same problem.

console log below 

PC1> ping 192.168.32.2

84 bytes from 192.168.32.2 icmp_seq=1 ttl=64 time=0.419 ms
84 bytes from 192.168.32.2 icmp_seq=2 ttl=64 time=0.609 ms
84 bytes from 192.168.32.2 icmp_seq=3 ttl=64 time=0.598 ms
84 bytes from 192.168.32.2 icmp_seq=4 ttl=64 time=0.524 ms
84 bytes from 192.168.32.2 icmp_seq=5 ttl=64 time=0.546 ms

PC1> arpConnection closed by foreign host.

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
